trtools/utils/tr_harmonizer.py

```diff
@@ -1091,19 +1091,16 @@ def GetLengthGenotypes(self) -> Optional[np.ndarray]:
         idx_gts = self.GetGenotypeIndicies()
         if idx_gts is None:
             return None
+        else:
+            idx_gts = idx_gts.astype(int)
```

Comment:
Why is this needed here? Were these floats, and using them as indices was failing before the cast? If so, should we move this line into the GetGenotypeIndicies function? Or do you believe it better to leave here?

Reply:
yes, exactly. They were `float64`s before. I'm not sure why...

I originally didn't make this change in `GetGenotypeIndicies` because I wasn't sure whether it would break anything. But the tests seem to pass after doing it in 1b43aa397e114f82b77eb11508f3c74f77254802, so I guess it's ok?